Sources/BijectiveDictionary/Documentation.docc/Using a BijectiveDictionary.md

```diff
@@ -0,0 +1,72 @@
+# Using a BijectiveDictionary
+Learn how to mutate and maintain a BijectiveDictionary.
+
+## Reserving Capacity
+Just like a `Dictionary`, if a ``BijectiveDictionary`` overflows its capacity, then it must internally allocate for more memory. While this process is automatic, it is not free. If we know the required capacity ahead of time, then we can allocate it in advance, thus eliminating unnecessary work. This can be done during initialization. 
+
+```swift
+let bDict = BijectiveDictionary<String, Int>(minimumCapacity: 10_000)
+```
+
+Or it can also be done during the lifetime of the dictionary. 
+
+```swift
+print(bDict.capacity) // 10_000
+bDict.reserveCapacity(200_000)
+print(bDict.capacity) // 200_000
+```
+## Reading a Value
+```swift
+let bDict: BijectiveDictionary = ["A": 1, "B": 2, "C": 3]
+bDict[left: "B"] // 2
+bDict[right: 3] // "C"
+```
+
+## Order
+># `BijectiveDictionary` does not preserve or guarantee ordering. 
+>Do not forget that just like `Dictionary`, a `BijectiveDictionary` is unordered. The order of key-value pairs is stable between mutations but is otherwise unpredictable. For more info, see the docs for [Dictionary](https://github.com/swiftlang/swift/blob/28c6cc105da2e917416002df3e0accfc2ad2d23f/stdlib/public/core/Dictionary.swift#L316).
```

Comment:
This is really good to mention. This is one thing that is somewhat buried in the official docs.